docs/rst/recording/usage/configuration.rst

```diff
@@ -374,15 +395,15 @@ In some applications, it may be required to use the ``publishTime`` as ``logTime
 Max Reception Rate
 ^^^^^^^^^^^^^^^^^^
 
-Limits the frequency [Hz] at which samples are processed, by discarding messages received before :code:`1/max-reception-rate` seconds have elapsed since the last processed message was received.
-When specified, ``max-reception-rate`` is set for all topics without distinction, but a different value can also set for a particular topic under the ``qos`` configuration tag within the builtin-topics list.
+Limits the frequency [Hz] at which samples are processed, by discarding messages received before :code:`1/max-rx-rate` seconds have elapsed since the last processed message was received.
```

Comment:
The description of the Max Reception Rate contains the note explaining where they can be set. In any case, I'll mark this as a TODO as it will make sense to always set the Topic QoS under the `qos` tag.